src/main/java/bookface/logic/commands/Command.java

```diff
@@ -7,6 +7,7 @@
  * Represents a command with hidden internal logic and the ability to be executed.
  */
 public abstract class Command {
+    public static final String MESSAGE_USAGE = "Get the program usage instructions by typing help!";
```

Comment:
When does this message usage appear?

Reply:
About the only time it pops up is when the users types whitespace only as input